calculator.rb

```diff
@@ -0,0 +1,64 @@
+def operator_list
+  puts "add(+)"
+  puts "subtract(-)"
+  puts "multiply(*)"
+  puts "divide(/)"
+  puts "exponify(^)"
+  puts "modulate(%)"
+end
+
+def calculator(num1, o, num2)
+  case o
+  when "+", "add"
```

Comment:
Convention dictates that inside `case` statements, we indent the `when` keywords.